websites/C/Chia Sẻ Nhạc/presence.ts

```diff
@@ -0,0 +1,156 @@
+const presence = new Presence({
+    clientId: "927785613309116476"
+  }),
+  browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+      largeImageKey: "lg",
+      startTimestamp: browsingTimestamp
+    },
+    path = document.location.pathname,
+    [buttons, timestamps, cover] = await Promise.all([
+      presence.getSetting<boolean>("buttons"),
+      presence.getSetting<boolean>("timestamps"),
+      presence.getSetting<boolean>("cover")
+    ]);
+  if (path.startsWith("/hd")) {
+    if (path === "/hd/video.html") presenceData.details = "Đang tìm video";
+    else if (path.endsWith("v-video.html"))
+      presenceData.details = "Đang tìm video Việt Nam";
+    else if (path.endsWith("us-video.html"))
+      presenceData.details = "Đang tìm video Âu Mỹ";
+    else if (path.endsWith("c-video.html"))
+      presenceData.details = "Đang tìm video Hoa";
+    else if (path.endsWith("k-video.html"))
+      presenceData.details = "Đang tìm video Hàn";
+    else if (path.endsWith("l-video.html"))
+      presenceData.details = "Đang tìm video Live";
+    else if (path.endsWith("h-video.html"))
+      presenceData.details = "Đang tìm video hài";
+    else if (path.endsWith("j-video.html"))
+      presenceData.details = "Đang tìm video Nhật";
+    else if (path.endsWith("f-video.html"))
+      presenceData.details = "Đang tìm video Pháp";
+    else if (path.endsWith("o-video.html"))
+      presenceData.details = "Đang tìm video nước khác";
+    else {
+      let paused = !document
+        .querySelector<HTMLDivElement>(
+          "#csnplayer > div.jw-controls.jw-reset > div.jw-display.jw-reset > div > div > div.jw-display-icon-container.jw-display-icon-display.jw-reset > div"
+        )
+        .ariaLabel.includes("Pause");
+      presenceData.details = document.querySelector<HTMLHeadingElement>(
+        "body > section > div.container > div > div.col-md-9 > div:nth-child(4) > div.col-md-4 > div > div.card-body > h2"
+      ).textContent;
+      presenceData.state = document
+        .querySelector<HTMLLIElement>(
+          "body > section > div.container > div > div.col-md-9 > div:nth-child(4) > div.col-md-4 > div > div.card-body > ul > li:nth-child(1)"
+        )
+        .textContent.slice(7);
+      if (cover) {
+        delete presenceData.largeImageKey;
```

Comment:
You do not need to delete it, the original value is automatically replaced

Reply:
Oh interesting, I'll fix it ASAP

Reply:
Done

Reply:
Still here

Reply:
Brainfart 💀 
Done fr this time